external/coreclr/project.json.template

```diff
@@ -3,7 +3,8 @@
     "{TFM}": {
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.2.0-beta-24721-02",
-        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02"
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02",
+        "runtime.native.System.Data.SqlClient.sni": "4.4.0-beta-24814-01",
```

Comment:
I would like to understand why it is needed at all. Is it just for the native shim? Why wouldn't this just be the live built shim?